cmd/dosa/util.go

```diff
@@ -114,16 +117,35 @@ func strToFieldValue(t dosa.Type, s string) (dosa.FieldValue, error) {
 	}
 }
 
-func printResult(res []map[string]dosa.FieldValue) {
-	if len(res) == 0 {
-		fmt.Println("Error: not found")
-		return
+func printResults(results []map[string]dosa.FieldValue) error {
+	w := tabwriter.NewWriter(os.Stdout, 0, 0, 3, ' ', tabwriter.Debug|tabwriter.StripEscape)
+	if len(results) == 0 {
+		return errors.New("Empty results")
 	}
-	fmt.Println("Results:")
-	for _, fieldValues := range res {
-		for field, value := range fieldValues {
-			fmt.Printf("%s: %v\n", field, reflect.Indirect(reflect.ValueOf(value)))
+	idx := 0
+	width := len(results[0])
+	keys := make([]string, width)
+	for k := range results[0] {
```

Comment:
Found 1 issue from my tests here when there's empty (null) value for the results:
e.g. the table has three columns (c1, c2, c3), we insert two rows: 

row1: (r1c1, nil, r1c3)
row2: (r2c1, r2c2, r2c3)

Two possibilities:
1. If row1 is results[0], then the CLI will only show columns c1 and c3
2. If row2 is results[0], then the CLI will show columns c1, c2 and c3, but row1's c2 value will be: <invalid reflect.Value>

Suggestions:
1. For printResults function, you can pass the list of fields to show (add fields to input signature). You can get the fields from: https://github.com/uber-go/dosa/blob/master/cmd/dosa/query.go#L92
2. We check if result[key] is valid. If not, we print 'nil' for it like cqlsh does.